pages/work/jebbit.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="section flex flex-wrap justify-center">
+    <div class="max-w-2xl w-full">
+      <div class="p-4 lg:p-12">
+        <div class="text-xs">
+          Case Study
+        </div>
+
+        <h1 class="text-4xl lg:text-6xl">
+          Jebbit
+        </h1>
+
+        <p>
+          Jebbit is an enterprise software platform that enables a future of
+          total data transparency between brands and consumers. Jebbit’s mission
+          is to make every brand-and-consumer interaction assumption free by
+          asking consumers explicitly and directly about their interests,
+          motivations, and preferences.
+        </p>
+      </div>
+
+      <div class="bg-black flex flex-wrap h-64 md:h-140 justify-center items-center">
+        <div class="w-1/2">
+          <JebbitLogo class="fill-white h-auto w-full" />
+        </div>
+      </div>
+
+      <div class="p-4 lg:p-12">
+        <h4>Challenge</h4>
+
+        <p>
+          Jebbit had several years of user feedback and data on the difficulty
+          in using the Campaign Builder product and sought to create a
+          lightweight version that is more accessible and easier to get started
+          for new customers.
+        </p>
+
+        <div class="bg-grey-light flex justify-center lg:flex-1">
+          <div class="pl-4 pr-4 pt-4 lg:pl-24 lg:pr-24 lg:pt-12">
+            <img
+              alt="Jebbit Experience Builder Legacy"
+              class="flex"
+              src="/img/work/jebbit/jebbit-legacy-builder.png"
+            >
+          </div>
+        </div>
+      </div>
+
+      <div class="p-4 lg:p-12">
+        <h4>Solution</h4>
+
+        <p>
+          The Ship Shape team began by creating a flexible grid layout to
+          maximize usability across various screen sizes and give the
+          application more familiar user patterns. In order to reduce the
+          complexity of creating campaigns in the Jebbit system, some custom
+          components were created to obfuscate those flows so that the user
```

Comment:
It's not immediately clear what is meant by this sentence. If I am understanding correctly, the gist is the flows are simplified and some of the complexity has been removed and handled behind the scenes, right? I think we could maybe break this into a couple sentences and simplify the language.